isis/src/kaguya/apps/mimap2isis/mimap2isis.cpp

```diff
@@ -78,13 +85,19 @@ namespace Isis {
   instXlater.Auto(otherLabels);
 
   PvlKeyword processId = label.findKeyword("PROCESS_VERSION_ID");
+  PvlKeyword processVersion = label.findKeyword("PRODUCT_VERSION_ID");
```

Comment:
rename this variable to productVersion so that it better matched the original keyword and is less confusing with processId right next to it.